src/guide/render-function.md

```diff
@@ -240,6 +240,45 @@ render() {
 }
 ```
 
+## Creating Component VNodes
+
+To create a VNode for a component, the first argument passed to `h` should be the component itself:
+
+```js
+render() {
+  return Vue.h(ButtonCounter)
+}
+```
+
+If we need to resolve a component by name then we can call `resolveComponent`:
```

Comment:
When are some use cases for resolving a component by name?

You mention later that it's often unnecessary, so as a reader, I'm curious to know a scenario where knowing this helper method would be important.

Reply:
I've updated the next few sentences to try to clarify. I struggled to find a wording that wasn't a clumsy mess. It still doesn't flow as cleanly as I'd like but hopefully it's clear enough.